### ISSUE TITLE
Is support the image zoom?

### ISSUE THREAD

**Issue by hengchengfei** [eventtus/photo-editor#25]
I want to zoom the image and then edit it.

How can I do it?

**Comment by halilyuce** [comment]
yeah its important, is it possible add zoom feature like instagram stories ? We cant drag and drop stickers and text outside of images :/
